dashboard/client/src/api/functionsApi.test.js

```diff
@@ -70,4 +71,26 @@ describe('functionsApi', () => {
       expect(second.name).toEqual(functionResponse1.name);
     });
   });
+
+
+  describe('replaceFunctionEnding', () => {
+    it('replaces function with trailing slash', async () => {
```

Comment:
Can we have a test that does nothing, i.e. it's not required?

Reply:
The function will always append the function name if the url doesn't end with `function`. Assuming that if a user has forgotten to put `/function` then they want the url to be `/<user-function-name>`